### DISCUSSION TITLE
Start a dryer using tasmota when enough PV power is available (delayed activation of usb after main relais is switched on)

### DISCUSSION THREAD

**Discussion by JanFellner** [arendst/Tasmota#15081] [Q&A] [answered]
Hi,

i am using tasmota since quite a while in combination with openwb to control devices if i have enough power from the sun to do stuff.
One thing which i kind of fail to get working is my dryer. It is not automatically starting if i give it power. I need to press a button.

I have a Gosund SP112 device which has additional two USB Ports which are controllable by tasmota.

How can i configure that if relais 1 is switched on that relais 2 makes a pulse e.g. 5 seconds after having powered on relais 1?

If that would be somehow possible i could press the start button using the USB connectors of the smart plug.
(ofc using an additional 5V relais or an optocoupler.)

Jan

**Comment by lalo-uy** [comment, marked as answer]
A rule triggered when power1 goes one seems to solve the problem.
Start a rule timer for some seconds to let the drayer to boot, an the tourn
on R2, with 5 sec pulsetime.
